data-generation/src/main/scala/thesis/Landy.scala

```diff
@@ -0,0 +1,36 @@
+package thesis
+
+import org.apache.spark.graphx.{Edge, EdgeRDD, EdgeTriplet, Graph, VertexRDD}
+import org.apache.spark.rdd.RDD
+import thesis.LTSV.Attributes
+import thesis.SnapshotDeltaObject.LandyAttributeGraph
+
+import java.time.Instant
+
+case class LandyVertexPayload(id: Long, validFrom: Instant, validTo:Instant, attributes:Attributes)
+case class LandyEdgePayload(id: Long, validFrom: Instant, validTo:Instant, attributes:Attributes)
+
+
+class Landy(val graph: LandyAttributeGraph) extends TemporalGraph[LandyVertexPayload, LandyEdgePayload] {
+
+  override val vertices: VertexRDD[LandyVertexPayload] = graph.vertices
+  override val edges: EdgeRDD[LandyEdgePayload] = graph.edges
+  override val triplets: RDD[EdgeTriplet[LandyVertexPayload, LandyEdgePayload]] = graph.triplets
+
+  override def snapshotAtTime(instant: Instant): Graph[LandyVertexPayload, LandyEdgePayload] = {
```

Comment:
Typen til snapshotAtTime for Snapshotdelta og Landy burde vel være den samme? Eller er den faktisk lik siden vi gjør litt generics magi

Reply:
Tror generics-magien kommer til unnsetning ja!